src/components/home.tsx

```diff
@@ -1,22 +1,33 @@
 import {Component, FC} from 'react'
 import templatesConfig from '../templates/templates_config'
 import { ListGroup } from 'react-bootstrap';
+import { Link } from "react-router-dom";
+
 
 const Home: FC = () => {
     
   const templates = Object.keys(templatesConfig).map((key) => (
     <ListGroup.Item key={key}>
-      <a href={`/app/${key}`}>
+      <Link to={`/app/${key}/job1`}>
```

Comment:
Could you tell me if this /job1 is necessary or not here, because that part should be linked to /app/${key} and there will be a list of jobs inside (right now in the Jobs View issue). When clicking the the job it should then link to `/app/${key}/job1`.

I guess my point is that it should not link to job1 directly from Home.

Reply:
I appended `job1` to the url so I could navigate to `/app/qa_hpwh/job1` from the main nav page. I can remove this, but as I note in the PR notes, one will not be able to enter a URL like `https://pnnl.github.io/Remote-QA-Web-App/app/qa_hpwh/job1` in their browser to navigate to a job. ALL navigation must be done via. react. 

If there is a page that maintains a list of jobs with ability to perform CRUD operations against that list, I think we'll be fine.